### PR TITLE
Add optional presubmit prow job for GCE PD CSI Driver with csi-proxy v1

### PR DESCRIPTION
**Pull request by mauriciopoppe**
/cc @jingxu97

### CODE REVIEW

**Comment by mauriciopoppe** [comment]
/hold

The version should have the suffix `gke`, we will wait until the binary is available

**Comment by mauriciopoppe** [comment]
/hold 

@jingxu97 this was an optional job that was supposed to run before merging https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/738, it could still be useful though if we decide to run the presubmit integration job from GCE PD CSI driver with this change before we make changes to https://github.com/kubernetes/kubernetes/blob/ce8b7afc8af35a08adbb88509cd8bac4ea6e0348/cluster/gce/config-common.sh#L167

File: config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml [hunk 1]
```diff
@@ -364,3 +364,50 @@ presubmits:
...
+        - --test=false
+        - --test-cmd=$GOPATH/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/run-windows-k8s-integration.sh
+        - --timeout=180m
+        env:
+        - name: WINDOWS_NODE_OS_DISTRIBUTION
+          value: "win2019"
+        - name: PREPULL_YAML
+          value: "prepull-head.yaml"
+        - name: CSI_PROXY_VERSION
+          value: "v1.0.0-rc.1"
```
Comment (jingxu97) [inline config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml:405]: we can change it to v1.0.0?

Comment (mauriciopoppe) [reply config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-windows.yaml]: changed to v1.0.0

**Comment by mauriciopoppe** [comment]
We no longer need this as PD CSI is already using csi-proxy v1
/close

**Pull request closed without merging**